### PR TITLE
Updates for dbt2 appimage and terraform 1.5.x

### PR DESCRIPTION
**Pull request by RDiMag**
These updates were required to get the benchmark to run.

### CODE REVIEW

File: aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml [hunk 1]
```diff
@@ -15,13 +15,15 @@
...
+         pgsql
       environment:
-        # Use fake PGDATA here in order to make dbt2-pgsql-build-db working
-        # with -r
-        DBT2PGDATA: /tmp
-        DBT2DBNAME: "{{ infra.servers.biganimal.dbt2.tags.dbname }}"
-        PGPASSWORD: "{{ pg_password }}"
+        # # Use fake PGDATA here in order to make dbt2-pgsql-build-db working
+        # # with -r
+        # DBT2PGDATA: /tmp
```
Comment (mw2q) [inline aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml:24]: i believe these can be removed.  did we remove them in the aurora test?

File: aws-dbt2-biganimal/04_execute/playbook-dbt2-run.yml [hunk 2]
```diff
@@ -46,23 +46,20 @@
...
+            --dbaas
+            --districts 1
+            --duration={{ dbt2_duration }}
+            --stats
+            --warehouses={{ dbt2_warehouse }}
+            pgsql
+            /tmp/dbt2-data
       environment:
-        DRIVER3: "1"
+        # DRIVER3: "1"
```
Comment (mw2q) [inline aws-dbt2-biganimal/04_execute/playbook-dbt2-run.yml:62]: go ahead and delete this line

**Review by mw2q — CHANGES_REQUESTED** [review]

File: aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml (hunk 1 above)
Comment (RDiMag) [reply aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml:24]: Yes, in aurora we have only PGPASSWORD and PGHOST, while RDS has only PGHOST.
```
    environment:
        PGPASSWORD: "{{ pg_password }}"
        PGHOST: "{{ infra.servers.aurora.postgres.public_ip }}"
```

Comment (mw2q) [reply aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml:24]: interesting.  i was pretty sure we set the password in the .pgpass file, so we don't need to set it here.  do you know how to check the .pgpass file?

Comment (RDiMag) [reply aws-dbt2-biganimal/03_prepare/playbook-dbt2-build-db.yml:24]: I haven't found a .pgpass file yet, but there is a dbt2- configure task in Ansible which creates and saves a pg password, so I think you are correct. I don't think we need pg_password here and will remove it.

**Review by bryan-bar — APPROVED** [review]

**Pull request merged**